### ISSUE TITLE
Pebble 2 HR paring issue

### ISSUE THREAD

**Issue by bbsen666** [Freeyourgadget/Gadgetbridge#702]
hi,

I am new to gadgetbridge. 

I have a pebble 2 HR with latest 4.3 firmware, I installed the latest version of gadgetbridge from F droid as suggested here.
My android phone is running OS 6.0.1

I activated my GPS and bluetooth and managed to find my pebble in gadgetbridge app. but after I granted permission for pairing in my pebble (tick) but my gadgetbridge app stuck in pairing. I went back one step and tried to pair again, it saidalready bonded with pebble xx xx xx, connecting.

Any advise on this will be very much appreciated. 

Thanks in advance.

**Comment by ashimokawa** [comment]
You should be inside the Bluetooth menu on the pebble while pairing. Also first confirm the dialog on the phone , then on the watch

**Comment by bbsen666** [comment]
Thank you for your reply, I did exactly what you said before still no luck.
Keep connecting....

**Comment by ozdreamern** [comment]
Do you also have the official Pebble app installed?  My experience was that this makes it difficult for GadgetBridge to establish and maintain connections.

**Comment by ashimokawa** [comment]
@bbsen666 
Which phone model do you have?
And can you send logs?
You could also try to force a low mtu (23) in pebble settings

**Comment by bbsen666** [comment]
Thanks again for all the feedback. Really appreciated.
As Mr ozdreamern said, gadgetbridge connects with my pebble smoothly after I uninstalled my pebble official app.
I managed to installed few downloaded pbw files as suggested in wiki but there are few apps I couldn't install properly. 
I granted access to notification for gadget bridge but I still couldn't receive any message notifications.

I have 2 problems with the official pebble app, it fails to reconnect automatically and when other people send me multiple messages, i only can receive first whatsapp notification, the rest are not shown. 

I am using LeEco Le2 on android 6.0.1.
